marklogic-data-hub-central/ui/src/config/messages.config.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const AdvancedSettingsMessages = {
+    'targetPermissions': {
+        'incorrectFormat' : (
+        <span>The format of the string is incorrect. The required format is 
+            <span style={{fontFamily: "monospace"}}> role, capability, role, capability,....</span> 
+        </span>),
+        'invalidCapabilities' : (
+        <span>The string contains invalid capabilities. Capabilities must be 
+            <span style={{fontFamily: "monospace"}}> read, insert, update, or execute</span>.
```

Comment:
@devinnicholson, the invalidCapabilities monospaced text needs to be updated slightly based on what Eleanor provided, it should look like the following:

<span style={{fontFamily: "monospace"}}>read</span>, <span style={{fontFamily: "monospace"}}>insert</span>, <span style={{fontFamily: "monospace"}}>update</span>, or <span style={{fontFamily: "monospace"}}>execute</span>.